examples/config-openid.json

```diff
@@ -7,6 +7,7 @@
   "http": {
     "address": "127.0.0.1",
     "port": "8080",
+    "externalUrl": "http://127.0.0.1:8080",
```

Comment:
Should this be "externalAddress", can we not detect and set the http scheme `http` or `https`?

Reply:
Let's not detect. I'm thinking this could be the address of a load balancer, or some kubernetes node.